src/platform/user/profile/vet360/actions/transactions.js

```diff
@@ -25,151 +26,199 @@ export const VET360_TRANSACTION_UPDATE_FAILED =
 export const VET360_TRANSACTION_CLEARED = 'VET360_TRANSACTION_CLEARED';
 export const VET360_CLEAR_TRANSACTION_STATUS =
   'VET360_CLEAR_TRANSACTION_STATUS';
-
-export function clearTransactionStatus() {
-  return {
-    type: VET360_CLEAR_TRANSACTION_STATUS,
-  };
-}
-
-export function fetchTransactions() {
-  return async dispatch => {
-    try {
-      let response;
-      if (isVet360Configured()) {
-        response = await apiRequest('/profile/status/');
-      } else {
-        response = { data: [] };
-        // Uncomment the line below to simulate transactions being processed during initialization
-        // response = localVet360.getUserTransactions();
-      }
-      dispatch({
-        type: VET360_TRANSACTIONS_FETCH_SUCCESS,
-        data: response.data,
-      });
-    } catch (err) {
-      // If we sync transactions in the background and fail, is it worth telling the user?
+export const ADDRESS_VALIDATION_CONFIRM = 'ADDRESS_VALIDATION_CONFIRM';
+export const ADDRESS_VALIDATION_ERROR = 'ADDRESS_VALIDATION_ERROR';
+
+export const clearTransactionStatus = () => ({
+  type: VET360_CLEAR_TRANSACTION_STATUS,
+});
+
+export const fetchTransactions = () => async dispatch => {
```

Comment:
I know this code was already there, but this seems like a good place to use `redux-thunk` in the future assuming we already have the infrastructure for it.